app/src/main/java/com/orange/ouds/app/ui/tokens/TokenPropertyIllustration.kt

```diff
@@ -59,7 +59,7 @@ private val defaultIllustrationSize = 64.dp
 @Composable
 fun IllustrationBox(
     modifier: Modifier = Modifier,
-    backgroundColor: Color = OudsColorKeyToken.Background.Emphasized.value,
+    backgroundColor: Color = OudsColorKeyToken.Surface.Status.Neutral.Muted.value,
```

Comment:
Background is different for spaces in Figma. As discussed together, this should be checked by Anton.